fitbenchmarking/fitbenchmarking
# 265 simple tests fb

#### Description of Work

Addresses part of #265 by duplicating the simple tests in NIST format for Mantid functions.

Also removed some simple tests that didn't add anything to the range of test cases.

#### Testing Instructions

1. Run `simple_tests` for some minimizers
2. Check results are as expected


Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
